src/Console/DatabaseStructure.php

```diff
@@ -61,6 +62,7 @@ protected function getHelp()
 Options
     -h|--help|-?       Show help information
     -v                 Show more debug information.
+    -e|--execute       Execute the dropping.
```

Comment:
Since this subcommand has a specific workflow with a unique parameter, I’d rather you create a separate console command that would be dry-run by default and suggest to use the -e flag to actually perform the destructive operations.

Reply:
I think it belongs to this "structure" family of commands.

Reply:
In a larger sense it does but it has a completely different goal and usage. To me it’s more like a cleanup task that is also about the database structure.

Reply:
I made some changes to the help text since we already had the situation that several options had only been valid for some commands.